pkgs/development/python-modules/x256/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "x256";
+  version = "0.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "00g02b9a6jsl377xb5fmxvkjff3lalw21n430a4zalqyv76dnmgq";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Find the nearest xterm 256 color index for an RGB.";
```

Comment:
You should read https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes completely: *Don’t include a period at the end.*

Reply:
sorry about that i will give it a through read